tests/Reports/ElementTypeReportTest.php

```diff
@@ -1,55 +0,0 @@
-<?php
-
-namespace DNADesign\Elemental\Tests\Reports;
-
-use DNADesign\Elemental\Reports\ElementTypeReport;
-use DNADesign\Elemental\Tests\Src\TestElement;
-use DNADesign\Elemental\Tests\Src\TestPage;
-use DNADesign\Elemental\Tests\Src\TestUnusedElement;
-use SilverStripe\Dev\FunctionalTest;
-use SilverStripe\ORM\ArrayList;
-use SilverStripe\View\ArrayData;
-
-class ElementTypeReportTest extends FunctionalTest
-{
-    protected $usesDatabase = true;
-
-    protected static $extra_dataobjects = [
-        TestElement::class,
-        TestPage::class,
-        TestUnusedElement::class,
-    ];
-
-    public function testReportShowsBlockTypes()
-    {
-        $this->logInWithPermission('ADMIN');
-
-        $result = (string) $this->get('admin/reports/show/DNADesign-Elemental-Reports-ElementTypeReport')->getBody();
-
-        $this->assertContains('Content Type', $result, 'Table has headings');
-
-        $this->assertContains('Unused Element', $result, 'Unused elements are still shown in the report');
```

Comment:
I think covering this in your new behat replacement is important. It doesn't appear to be there, or have I missed something?

Reply:
Yes you're right in that this isn't in the new behat test. The stub it's checking is for phpunit, we'd need to make some stub elements available to the behat tests to make this work. Let's raise a new issue for it

Reply:
Ideal 👍